docs/assets/js/script.js

```diff
@@ -24,7 +24,7 @@ devicon.controller('IconListCtrl', function($scope, $http, $compile) {
   });
 
   var versionStr = '@' + $scope.latestReleaseTagging;
-  var baseUrl = `https://cdn.jsdelivr.net/gh/${gitHubPath}${versionStr}/`
+  var baseUrl = `https://cdn.jsdelivr.net/gh/${gitHubPath}@a6e7612e7e105247eb1103fc3756c09e838b51a7/`
```

Comment:
this too

Reply:
fixed: 726d58f1bcadbd4da6910e041d0bf5a5d3ec092b